firebase-dataconnect/gradleplugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Why are you using your own wrapper for the included build?

Reply:
Ahh you're right that this extra copy of the gradle wrapper isn't required. I had never tried it without the gradle wrapp and assumed it was needed. I've since removed it.